### DISCUSSION TITLE
What happened to the todo list?

### DISCUSSION THREAD

**Discussion by mywalkb** [LSPosed/LSPosed#774] [Q&A] [answered]
In the first period after the fork, this project was fast.
Many items were created and immediately closed because they were made.
Now everything seems to have stopped.
There are about 8 items in TODO list from many days.
For example Module API, randmon package name and others, none is working on them. Why?
What happened?
EdXposed is dead, I hope this project alive.
Sure there is many fixes, but no new features.

**Comment by yujincheng08** [comment, marked as answer]
Those TODOs are not easy to be done. We are working on them. But not that fast.

**Reply by mywalkb** [reply]
Have you an ETA for Module API?
It's a very good improvement IMHO.
